pkg/workloadmeta/store.go

```diff
@@ -370,6 +368,27 @@ func (s *store) Reset(newEntities []Entity, source Source) {
 	s.Notify(events)
 }
 
+func (s *store) startCandidatesWithRetry(ctx context.Context) error {
```

Comment:
should we have a "Retry backoff" per candidate instead of a "global retry"

Reply:
I think that the complexity that it would add is not worth it at this point.